tasks.py

```diff
@@ -56,11 +58,17 @@ def jsonDecodeObjectHook(obj):
 	regionPrefix = _gaeApp.split("~")[0]
 	queueRegion = regionMap.get(regionPrefix)
 
-if not queueRegion:
+if not queueRegion and os.getenv("TASKS_EMULATOR") is None:
 	# Probably local development server
 	logging.warning("Taskqueue disabled, tasks will run inline!")
 
-taskClient = tasks_v2.CloudTasksClient()
+if os.getenv("TASKS_EMULATOR") is None:
```

Comment:
Can you change this condition to 
`if not utils.isLocalDevelopmentServer or os.getenv("TASKS_EMULATOR") is None: `?
This provides an additional safety check if the project is accidentally deployed with the TASKS_EMULATOR env var set in app.yaml